.github/workflows/ci_on_ubuntu.yml

```diff
@@ -252,40 +264,87 @@ jobs:
+  test_shell_espnet2:
+    runs-on: ${{ matrix.os }}
+    needs: process_labels
+    if: |
+      github.event.pull_request.draft == false
+    strategy:
+      max-parallel: 20
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+        pytorch-version: ["2.4.0"]
+        chainer-version: ["6.0.0"]
+        use-conda: [false]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ matrix.pytorch-version }}-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/Makefile') }}
 
-      - name: test utils
-        run: ./ci/test_utils.sh
-      - uses: codecov/codecov-action@v2
+      - name: Make environment
+        uses: ./.github/actions/prepare-environment
         with:
-          flags: test_utils
-      - name: coverage erase
-        continue-on-error: true
-        run: |
-          source tools/activate_python.sh
-          coverage erase
+          os-version: ubuntu
+          python-version: ${{ matrix.python-version }}
+          pytorch-version: ${{ matrix.pytorch-version }}
+          chainer-version: ${{ matrix.chainer-version }}
+          use-conda: ${{ matrix.use-conda }}
+      - name: test shell
+        run: ./ci/test_shell_espnet2.sh
 
-      - name: test espnetez integration
-        run: ./ci/test_integration_espnetez.sh
+  test_configuration_espnet2:
+    runs-on: ${{ matrix.os }}
+    needs: process_labels
+    if: |
+      github.event.pull_request.draft == false
+    strategy:
+      max-parallel: 20
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+        pytorch-version: ["2.6.0"]
+        chainer-version: ["6.0.0"]
+        use-conda: [false]
+        task: [asr, asr_transducer, lm, tts, enh, ssl, enh_asr]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ matrix.pytorch-version }}-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/Makefile') }}
+      - name: Make environment
+        uses: ./.github/actions/prepare-environment
+        with:
+          os-version: ubuntu
+          python-version: ${{ matrix.python-version }}
+          pytorch-version: ${{ matrix.pytorch-version }}
+          chainer-version: ${{ matrix.chainer-version }}
+          use-conda: ${{ matrix.use-conda }}
+      - name: Clean environment
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf /root/.cache/*
+      - name: test configuration
+        run: ./ci/test_configuration_espnet2.sh ${{ matrix.task }}
       - uses: codecov/codecov-action@v2
         with:
-          flags: test_integration_espnetez
-
+          flags: test_configuration_espnet2_${{ matrix.task }}
 
   test_import:
```

Comment:
This test is making CI fail. Can you comment it? I will need to modify it later. 
Especially, with the new changes for ESPnet2 installation, it requires moving some files between directories during the installation process.  I will be activating it later